packages/frontend-near-wallet/src/modules/wallet-migration/components/SelectDestinationWallet.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { styled } from '../../../styles';
+import IconWallet from '../../../assets/images/IconWallet';
+
+import Button from '../../../components/Button';
+import IconLedger from '../../../assets/images/IconLedger';
+import ImgMyNearWallet from '../../../assets/images/ImgMyNearWallet';
+
+
+const WALLET_OPTIONS = [
+    {
+        id: 'my-near-wallet',
+        name: 'My NEAR Wallet',
+        icon: <ImgMyNearWallet />,
+    },
+    {
+        id: 'ledger',
+        name: 'Ledger',
+        icon: <IconLedger />,
+    },
+];
+
+
+const SelectDestinationWallet = ({ handleCancel, handleSetWalletType, handleTransferMyAccounts, walletType }) => {
+    return (
+        <StyledContainer>
+            <IconWallet />
+            <h4 className='title'>Select a wallet to transfer accounts</h4>
+            <WalletOptionsListing>
+                {WALLET_OPTIONS.map(({ id, name, icon }) => {
+                    const isSelected = id === walletType;
+                    return <WalletOptionsListingItem
+                        className={`${isSelected ? 'active' : ''}`}
+                        onClick={() => handleSetWalletType(id)}
+                        key={name}
+                    >
+                        <h4 className='name'>{name}</h4>
+                        {icon}
+                    </WalletOptionsListingItem>;
+                })}
+            </WalletOptionsListing>
+            <ButtonsContainer>
+                <StyledButton onClick={handleCancel} variant="gray">
+                    Cancel
+                </StyledButton>
+                <StyledButton onClick={handleTransferMyAccounts}>
```

Comment:
This is broken for ledger. It proceeds with MNW secret page and migration even if you select ledger

Reply:
Yeah. There's no support for ledger here yet